packages/javascript-sdk/src/token-manager/index.ts

```diff
@@ -22,6 +22,7 @@ import { tokensWillExpireWithinThreshold } from './helpers';
 interface GetTokensOptions extends ConfigOptions {
   forceRenew?: boolean;
   login?: 'embedded' | 'redirect' | undefined;
+  skipBackgroundRequest?: boolean;
```

Comment:
I think we should JS Doc comment this `GetTokensOptions` interface so that the IDE gets more intellisense on this type. Especially this new addition.

Reply:
Done